utilities/include/vec3.hpp

```diff
@@ -1,120 +1,131 @@
+#ifndef VEC3.HPP
+#define VEC3.HPP
+
 //code by Ameya Vikrama Singh
```

Comment:
This line is not required. If you're worried about code ownership, you need not be: Git and GitHub store information about the author of files (and in fact, you can even find who wrote a specific line (even if the project has millions of lines of code), using `git blame`.)